lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/expectations.js

```diff
@@ -235,19 +235,23 @@ const expectations = [
             items: [
               {
                 url: 'http://localhost:10200/byte-efficiency/gzip.html',
+                finished: true,
               },
               {
                 url: 'http://localhost:10200/byte-efficiency/script.js?gzip=1',
                 transferSize: '1100 +/- 100',
                 resourceSize: '53000 +/- 1000',
+                finished: false,
               },
               {
                 url: 'http://localhost:10200/byte-efficiency/script.js',
                 transferSize: '53200 +/- 1000',
                 resourceSize: '53000 +/- 1000',
+                finished: true,
               },
               {
                 url: 'http://localhost:10200/favicon.ico',
+                finished: true,
```

Comment:
it's nice to have smoke coverage, but clearly we're not too worried about covering this audit's entire output given the assertions here, so this could be removed if it feels excessive

Reply:
I like the idea of asserting these *somewhere* so seems fine 🤷‍♂